### DISCUSSION TITLE
SubtitleEdit-3.6.11-Setup.exe infected

### DISCUSSION THREAD

**Discussion by llilitavi3** [SubtitleEdit/subtitleedit#6688] [General]
![image](https://user-images.githubusercontent.com/83832309/220794639-c348f210-5d57-4c04-9f20-4106a927442a.png)
downloaded and scan say is infected

**Comment by niksedk** [comment]
It's very common that unknown .exe files are flagged as a threat, Normally a new SE version will give 1-4 warnings from lesser known anti-virus companies.
SE previously used an .exe file packer, but I had to stop doing that, as SE gave 20+ warnings!

And as always, if you want to help, **please report "false positive" to the anti-virus provider**, thx :)
Then the anti-virus provider will take a closer look at the file, and after analyzing they will (normally) remove the threat warning after 2-10 days.
Note: Sometimes it can be a bit difficult and a lot of hassle to report false positives (probably a time consuming pain for many devs)

**Comment by niksedk** [comment]
Older issues with this:  https://github.com/SubtitleEdit/subtitleedit/issues?q=is%3Aissue+is%3Aclosed+virus

Especially this one: https://github.com/SubtitleEdit/subtitleedit/issues/5429#issuecomment-960499062

**Comment by niksedk** [comment]
OK, I reported the issue to SecureAge, so try again in a week or so: https://www.virustotal.com/gui/file-analysis/MWQwZGY3YjBiYjhjMzc4MjA3Mjc1MzgxYTBiNjcyMDM6MTY3ODAyMTIxOA==

````

Your submission (20230303-152600819900/SubtitleEdit-3.6.11-Setup-infected.zip) has been successfully processed. Your false detection(s) will be cleared within the next 72 hours; please notify us at [secureaplus@secureage.com](mailto:secureaplus@secureage.com) if that is not the case. For APEX results on VirusTotal, false detections may take additional time to clear. You may need to reanalyse files in order to get the updated detection result.

````

**Comment by niksedk** [comment]
@llilitavi3: try again :)

![image](https://user-images.githubusercontent.com/1261331/223557895-1ae59770-b080-4217-85da-a286c22096e1.png)
